src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -130,7 +130,7 @@ def acr_update_get(client,
     )
 
 def acr_update_custom(instance,
-                      admin_enabled=None,
+                      admin_enabled,
```

Comment:
This should still be admin_enabled=None because on updates, "None" means "leave this alone".

What you can do is:
```Python
register_cli_argument('acr create', 'admin_enabled', nargs='?', ..., choices=['true', 'false'])
register_cli_argument('acr update', 'admin_enabled', nargs='1', ..., choices=['true', 'false'])
```